ui/app/templates/allocations/allocation/task/index.hbs

```diff
@@ -0,0 +1,94 @@
+{{#global-header class="page-header"}}
+  <span class="breadcrumb">Allocations</span>
+  {{#link-to "allocations.allocation" model.allocation class="breadcrumb"}}
+    {{model.allocation.shortId}}
+  {{/link-to}}
+  {{#link-to "allocations.allocation.task" model.allocation model class="breadcrumb"}}
+    {{model.name}}
+  {{/link-to}}
+{{/global-header}}
+{{#gutter-menu class="page-body"}}
+  {{partial "allocations/allocation/task/subnav"}}
+  <section class="section">
+    <h1 class="title">
+      {{model.name}}
+      <span class="bumper-left tag {{model.stateClass}}">{{model.state}}</span>
+    </h1>
+
+    <div class="boxed-section is-small">
+      <div class="boxed-section-body inline-definitions">
+        <span class="label">Task Details</span>
+        <span class="pair">
```

Comment:
can try dl/dd here instead, if that makes the css any easier.

Reply:
Maybe. It would certainly be semantically better.